src/App.js

```diff
@@ -1,8 +1,37 @@
 import "./app.css";
+import { createElement } from "./utils/element";
 
-export function App() {
-  const app = document.createElement("div");
-  app.className = "app";
-  app.innerHTML = "<b>Awesome</b> Webpack Template!";
+export function createApp() {
+  const app = createElement("div", {
+    className: "app",
+    children: [
+      createElement("div", {
+        className: "time",
+        innerText: "60",
+      }),
+      createElement("button", {
+        classname: "startTimeButton",
+        innerText: "Go!",
+        onclick: () => {
+          createCountdownTimer();
+        },
+      }),
+    ],
+  });
+
+  // document.createElement("div");
+  // app.className = "app";
+  // app.innerHTML = "60";
+
+  function createCountdownTimer() {
+    const sec = 30;
+    const timer = setInterval(function () {
```

Comment:
`setInterval` returns an intervalId, `setTimeout` returns a timeoutId.
I would rename the variable.
```suggestion
    const intervalId = setInterval(function () {
```